leeanp.php

```diff
@@ -21,4 +22,5 @@
 require_once LEAN_WOOCOMMERCE_API_PLUGIN_DIR . 'vendor/autoload.php';
 
 // Run the plugin setup.
-new PluginSetup();
+$setup = new PluginSetup();
+register_activation_hook( __FILE__, array( $setup, 'maybe_deactivate' ) );
```

Comment:
Yeah sure, I just copied from the wp plugin, I didn't remember that syntax! haha Im on it, also with the others. Thank you!